### ISSUE TITLE
Support for authorization on kafka instance

### ISSUE THREAD

**Issue by csuryac** [sky-uk/kafka-configurator#57]
Does this support Kafka which is enabled with SASL . how do we pass the config file has authorization details

**Comment by r-glyde** [comment]
Yeah, this just uses the underlying kafka admin client so everything needed should be supported (we use this with SSL fine). Some examples [here](https://github.com/sky-uk/kafka-configurator#injecting-kafka-admin-client-config) but essentially you can either pass the property as a command line argument (e.g. `--properties security.protocol=SSL,ssl.keystore.password=bar`) or you can set via environment variables that are just the property name uppercased prefixed with `KAFKA_` (with `_` replacing `.`) e.g. `KAFKA_SECURITY_PROTOCOL=SSL`.

**Comment by nc-ruth** [comment]
@r-glyde do you know if kafka-configurator supports username + password (SASL/SCRAM)?
